README.md

```diff
@@ -254,22 +298,49 @@ Rspec.desrcibe TShirt do
+
+# Example structure
+
+When we write examples, we should try to structure the example so it tells a story. Good testing stories take this form:
+
+1. Setup
+1. Exercise
+1. Verify
+1. Teardown
+
+Or:
+
+1. Setup
+1. Exercise / Verify
+1. Teardown
+
+In a language like Ruby we can usually ignore Teardown. So lets just focus on Setup, Exercise, and Teardown. This is called a [Four-Phase Test and can be read about here](https://robots.thoughtbot.com/four-phase-test).
+
+## Setup
+
+This is where we create the testing fixtures and setup the system under test.
+
+## Exercise
+
+Use and exercise the system under test.
 
-This change did violate one Ruby tenant though. We've actually made the code less DRY (Don't Repeat Yourself) because we have repeated our `valid_attributes` (`{ size: "XS" }`) in the last test. This is a trade-off to this approach to testing. But it is much less of a trade-off compared to the pitfalls of our previous approaches.
+## Verify
 
-Nitro tests suffer greatly from issues of over using `before` and `let` helper methods. As do some learn.co labs. Understanding these method drawbacks puts you a fantastic position to help Nitro clean up its tests.
+Verify that the exercise step produced the expected behavior.
 
-## Many Features - What is Important
+# Many Features - What is Important
 
-`rspec` has quite a lot of [features and helper methods](https://relishapp.com/rspec/rspec-core/v/3-7/docs). Many of them are very convenient, and many are helpful. But worrying too much about the "syntactic sugar" around your tests is not the point. What makes good tests?
+`rspec` has quite a lot of features and helper methods. Many of them are very convenient, and many are helpful. But worrying too much about the "syntactic sugar" around your tests is not the point. What makes good tests?
 
 As you gain experience, you will form your own opinions on what kind of tests provide the most value. Here are some questions to keep in mind as you start writing your own tests:
 
-* Is the intent of my test clear?
+* Is the intent of my test clear? What is my subject? What am I describing, and how am I verifying that?
 * Am I coding only what is necessary for my test?
 * Does my test enforce expectations that must stay intact?
 
-Many developers have the mind set that simply having any tests is a step in the right direction. And they are right :).
+Many developers have the mind set that simply having any tests is a step in the right direction. The is wishful thinking. A poorly written can be slow, and brittle, and it might not verify what it says it will. The best way to learn what is the right test and how to wright them is to start trying it.
```

Comment:
```diff
- The is wishful thinking.
+ This is wishful tinking.
```

While I agree, this reads in a harsh tone - consider the value of this statement and perhaps omit it? I think the important message here is that specifications - too - are code. They have a maintenance cost. Like everything in business, we must be diligent to ensure that their benefit outweighs the cost.